tests/includes/run.sh

```diff
@@ -18,10 +18,11 @@ run() {
   # Prevent command from killing the script so we can capture its exit code
   # AND output. Also, make sure to grab both STDOUT and STDERR. We should be
   # using set -o pipefail here but that's unfortunately not supported by the shell.
+  set_verbosity
```

Comment:
I don't remember exactly why, but this was required to get the full output with the linters. Can you make a pass with a failing lint test (ineffassign in particular) and verify that we still see the output?

Reply:
That's fine, but we need to ensure we have all the flags set correctly and then we turn off the ones we don't want, in this case `set +e`